keyboards/ymdk/id75/rp2040/keyboard.json

```
@@ -0,0 +1,13 @@
+{
+    "manufacturer": "tao heihei",
```

For consistency with `f103` revision, unless this is intentional?
```suggestion
    "manufacturer": "YMDK",
```

---

I only ask because the `manufacturer` field is what will show up in USB device information, which I could potentially see causing confusion.